tensorflow/workspace.bzl

```diff
@@ -508,6 +508,14 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+    name = "lmdb",
```

Comment:
What has been done appears to be correct. The `licenses(["notice"])` statement is in the associated lmdb.BUILD file. Please note that `web_library_external` basically uses its licenses attribute to generate that statement, when it creates a BUILD file.